### PR TITLE
hotfix: added dark mode for profile page

### PR DESCRIPTION
**Pull request by PeerRich**
forgot to add dark mode for profile page. added.

image

### CODE REVIEW

File: components/Shell.tsx [hunk 1]
```diff
@@ -266,7 +270,7 @@ export default function Shell(props) {
       </div>
 
       <main className="-mt-32">
-        <div className="max-w-7xl mx-auto pb-12 px-4 sm:px-6 lg:px-8">{props.children}</div>
+        <div className="max-w-7xl mx-auto pb-12 px-4 sm:px-6 lg:px-">{props.children}</div>
```
Comment (emrysal) [inline components/Shell.tsx:273]: I don't think this was intended

**Review by emrysal — CHANGES_REQUESTED** [review]

File: components/Shell.tsx (hunk 1 above)
Comment (PeerRich) [reply components/Shell.tsx:273]: whops. good spot! changed

**Review by emrysal — APPROVED** [review]

File: components/Shell.tsx [hunk 2]
```diff
@@ -36,14 +36,18 @@ export default function Shell(props) {
 
   return session ? (
     <div>
-      <div className="bg-gradient-to-b from-blue-600 via-blue-600 to-blue-300 pb-32">
-        <nav className="bg-blue-600">
+      <div className="dark:from-gray-900 dark:to-gray-900  bg-gradient-to-b from-blue-600 via-blue-600 to-blue-300 pb-32">
+        <nav className="dark:bg-gray-900 bg-blue-600">
           <div className="max-w-7xl mx-auto sm:px-6 lg:px-8">
-            <div className="border-b border-blue-500">
+            <div className="border-b border-blue-500 dark:border-gray-">
```
Comment (baileypumfleet) [inline components/Shell.tsx:42]: Looks like you're missing the last part of the colour code (e.g. border-gray-**400**)

**Review by pumfleet — CHANGES_REQUESTED** [review]

File: components/Shell.tsx (hunk 2 above)
Comment (PeerRich) [reply components/Shell.tsx:42]: wtf. fixing .. thanks!

**Review by pumfleet — APPROVED** [review]

**Pull request merged**